README.md

```diff
@@ -109,6 +123,36 @@ You can run the application with this command: `air`.
 
 It will generate and reload thyra each time a file is modified.
 
+### Plugin installation
+
+Thanks to its modular architecture, it is possible to use third party plugins in Thyra.
+Plugins must be placed in a `plugins` folder in the Thyra config directory: `$HOME/.config/thyra`.
```

Comment:
also valid for macos

Reply:
I reused this because it is the folder already used to store the wallet files. But, indeed $HOME is not defined for windows... so wallets are stored at the root of the disk... (for example D:\.config\thyra)
I propose to put all of this in the user directory ( C:\Users\lucas\.config\thyra )
WDYT ? i don't know what are the best practices for windows.

Reply:
It's clearly better to change it to user directory. I don't know windows best practices on that ...